### ISSUE TITLE
Placeholders for dropdowns are visible only when editable

### ISSUE THREAD

**Issue by goyal-anjali** [microsoft/azuredatastudio#21408]
If the editable property is set to false, placeholder for dropdown is not visible, instead the first value is visible.
Is there a way where placeholder is visible without setting it to editable as -
1. CSS for editable makes the borders for dropdown red (image attached below)
2. The dropdown values can be edited when they are supposed to be fixed/constant

![image](https://user-images.githubusercontent.com/66509246/207295232-ce76f4fc-275a-4968-bff3-a8c1b0faa0ae.png)

**Comment by alanrenmsft** [comment]
@aasimkhan30 could you please take a look?

**Comment by goyal-anjali** [comment]
Following up on this issue

**Comment by aasimkhan30** [comment]
@alanrenmsft, I don't think we support placeholders for non-editable dropdown since there is an option always selected for it. 

@goyal-anjali, you can have a first option as 'Select a target platform' and then fail the validation if the user hasn't changed that. I will try to work on it for the next ads release.

**Comment by aasimkhan30** [comment]
Moving this to backlog as there are alternate ways of doing this and it is not a blocking issue.
